CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# SEED Version 2.13.0 Prerelease 1
+
+Date Range: 11/20/21 - 12/27/21
+
+Closed Issues and Features (Total: 17):
+- Fixed [#2585]( https://github.com/SEED-platform/seed/issues/2585 ), New feature notification on latest page visit after release
```

Comment:
Whoops, we should have added the "ignore" label to this -- it wasn't implemented we just closed it
```suggestion
```